index.js

```diff
@@ -148,6 +148,13 @@ function convert(obj) {
 
     partial[cls] = 0;
 
+    // handle numeric values
+    if(vAmount.match(/^[0-9.]+$/))
```

Comment:
Please use Google code style https://google-styleguide.googlecode.com/svn/trunk/javascriptguide.xml?showone=Code_formatting#Code_formatting to be consisted with the rest of code:
- one space after `if`
- opening curly brace on the same line as `if`
- one space before opening curly brace